CMakeLists.txt

```diff
@@ -184,6 +174,12 @@ endif()
 
 # Tests require Python3
 find_host_package(PythonInterp 3 REQUIRED)
+find_host_package(nosetests)
+
+find_package(SPIRV-Headers QUIET CONFIG)
+if(NOT SPIRV-Headers_FOUND)
+  find_package(SPIRV-Headers REQUIRED)
+endif()
```

Comment:
Sorry I do not know enough about cmake, to fully understand what will happen here.  Which locations for spirv-headers takes precedence?  In particular, if I have spirv-headers checked out in external, is that guaranteed to be the version that is found when I do `cmake <source_dir>`?

Reply:
Ah I mixed up the order, good catch. First it looks for an installed `SPIRV-HeadersConfig.cmake`, then it looks for the headers.

It should be the other way around.

Reply:
I changed the order and added a comment in the `FindSPIRV-Headers.cmake` and before these find package calls to clarify. Also, I explicitly do `find_package(MODULE)` now.

Also, `FindSPIRV-Headers.cmake` only checks `external` and `SPIRV-Headers_SOURCE_DIR` now and does not look in the sysroot (since installed headers should come with a CMake Config).